content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,140 @@
+
+**metadata.csv** - The file that lists and stores the metrics collected by each integration.
+
+**custom check** - If you have a unique system that you want to monitor, or if you're going to expand the metrics already sent by an integration, you can build a [custom check][11] to define and send metrics to Datadog. However, if you want to monitor a generally available application, public service, or an open source project, and we don't have an integration that already exists for it, you should consider [building a new integration][1] instead of a custom check.
+
+**tagging** - [Tags][12] are a way to add customization to metrics so that you can filter and visualize them in the most useful way to you.
+
+**logging** - If the system you are monitoring has logs, you can customize the logs you are sending to Datadog and use our [Logging Management solution][13] to manage and analyze them.
+
+## Setting up an integration
+
+The Datadog Agent package includes most integrations, so you don't need to download each individually.
+
+### API and Application keys
+
+In order to download the the Datadog Agent package, you need an [API and Application key][14]. You can manage your accounts API and Application keys in the [API Settings page][15] of the UI.
+
+### Installation
+
+If you want to connect with a cloud service provider, navigate to that provider on the [Integrations page][16] for specific instructions on how to connect. For most other integrations, start by installing the [Datadog Agent][17]. Most integrations are supported on our containerized agents: [Docker][18], and [Kubernetes][19]. After you've downloaded the Agent, go to the [Integrations page][16] section to find specific configuration instructions for individual integrations.
+
+### Configuration
+
+Configurations are more specifc to [individual integrations][16]. Configure most integrations by editing the `conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory. Most integration have a sample `conf.yaml` that lists all available configuration options.
+
+For example, this is a sample `conf.yaml` for the `http_check` integration:
+
+```
+init_config:
+
+instances:
+  - name: Example website
+    url: https://example.com/
+    # disable_ssl_validation: false      # default is true, so set false to check SSL validation
+    # ca_certs: /path/to/ca/file         # e.g. /etc/ssl/certs/ca-certificates.crt
+    # check_certificate_expiration: true # default is true
+    # days_warning: 28                   # default 14
+    # days_critical: 14                  # default 7
+    # timeout: 3                         # in seconds. Default is 10.
+  - name: Example website (staging)
+    url: http://staging.example.com/
+```
+
+### Validation
+
+To validate your configuration, [run the Agent's `status` subcommand][20], and look for new configuration under the Checks section.
+
+### Tagging
```

Comment:
Tagging should be before the validation section since validation is the last step of the process. 

In the case of Agent integrations, they can either define tags in the datadog.yaml file, and so those tags are applied to ALL integrations data. Or they can define tags in the corresponding integrations configuration file in order to apply tags to data only emitted by this particular integration.

Tags Environment variables for the Agent will apply tags to all Data. Container labeled with the dd tag label will have only its data tagged.